cz.zeropage.Formiko.json

```diff
@@ -0,0 +1,300 @@
+{
+    "app-id": "org.flatpak.Formiko",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "formiko",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
```

Comment:
What do you need session-bus for?

Reply:
Without that, i got error
```
Failed to register: 
 DBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
You can pass `--log-session-bus` to `flatpak run` to check what exactly you need.

Reply:
I really don't know, what all I must explicit allow. Formiko use webkit, so there could be more permission for it.

I found this: https://github.com/WebKit/webkit/blob/master/Tools/flatpak/org.webkit.WebKit.yaml which has `--socket=system-bus". 
```
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C3: -> org.freedesktop.DBus fake AddMatch for org.freedesktop.DBus.RequestName
C4: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus.RequestName
C5: -> org.freedesktop.DBus fake wildcarded AddMatch for cz.zeropage.Formiko
C6: -> org.freedesktop.DBus fake wildcarded AddMatch for org.gtk.vfs
C7: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.cz.zeropage.Formiko
C8: -> org.freedesktop.DBus fake ListNames
B1: <- org.freedesktop.DBus return from C1
B2: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B3: <- org.freedesktop.DBus return from C2
*SKIPPED*
B4: <- org.freedesktop.DBus return from C3
*SKIPPED*
B5: <- org.freedesktop.DBus return error org.freedesktop.DBus.Error.NameHasNoOwner from C4
*SKIPPED*
B6: <- org.freedesktop.DBus return from C5
*SKIPPED*
B7: <- org.freedesktop.DBus return from C6
*SKIPPED*
B8: <- org.freedesktop.DBus return from C7
*SKIPPED*
B9: <- org.freedesktop.DBus return from C8
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C10: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.UDisks2VolumeMonitor
C11: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.AfcVolumeMonitor
C12: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.MTPVolumeMonitor
C13: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C14: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.Daemon
C15: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C16: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.GoaVolumeMonitor
C17: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.mountpoint_dnssd
C18: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.mountpoint_2575
C19: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.GPhoto2VolumeMonitor
C20: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.mountpoint_2227
C21: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.mountpoint_http
C22: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.mountpoint_19836
C23: -> org.freedesktop.DBus fake GetNameOwner for org.gtk.vfs.Metadata
C24: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
C25: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
Filtering message due to arg0 cz.zeropage.formiko, policy: 0 (required 3)
*HIDDEN* (ping)
B10: <- org.freedesktop.DBus return from C9
*SKIPPED*
B11: <- org.freedesktop.DBus return from C10
*SKIPPED*
B12: <- org.freedesktop.DBus return from C11
*SKIPPED*
B13: <- org.freedesktop.DBus return from C12
*SKIPPED*
B14: <- org.freedesktop.DBus return from C13
*SKIPPED*
B15: <- org.freedesktop.DBus return from C14
*SKIPPED*
B16: <- org.freedesktop.DBus return from C15
*SKIPPED*
B17: <- org.freedesktop.DBus return from C16
*SKIPPED*
B18: <- org.freedesktop.DBus return from C17
*SKIPPED*
B19: <- org.freedesktop.DBus return from C18
*SKIPPED*
B20: <- org.freedesktop.DBus return from C19
*SKIPPED*
B21: <- org.freedesktop.DBus return from C20
*SKIPPED*
B22: <- org.freedesktop.DBus return from C21
*SKIPPED*
B23: <- org.freedesktop.DBus return from C22
*SKIPPED*
B24: <- org.freedesktop.DBus return from C23
*SKIPPED*
B25: <- org.freedesktop.DBus return from C24
*SKIPPED*
B26: <- (no sender) return from C25
*REWRITTEN*
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
```
C25: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
Filtering message due to arg0 cz.zeropage.formiko, policy: 0 (required 3)
```

It means that your application requests `cz.zeropage.formiko` (note the lower case). If you rename manifest file to lowercase and change app-id in the file, it will work fine.

Reply:
fixed, thanks